src/rpc/blockchain.cpp

```diff
@@ -2425,3 +2426,87 @@ UniValue getburnreport(const UniValue& params, bool fHelp)
 
     return json;
 }
+
+UniValue createmrcrequest(const UniValue& params, const bool fHelp) {
+    if (fHelp || params.size() > 2) {
+        throw runtime_error("createmrcrequest <force> <dry_run>\n"
+                            "\n"
+                            "<force> - If true, create the request even if it results "
+                            "in a reward loss or ban from the network. Defaults to false.\n"
```

Comment:
Having a force seems like it could lead to problems especially if someone meant to do a dry run but forgot an argument. What's the rationale for including this? Is this meant more for testing?

Reply:
Yes. This is not the final form for the rpc.

Reply:
The force will be removed after testing is done.